src/mangata-rococo.js

```diff
@@ -0,0 +1,161 @@
+
+/**
+ * README!
+ *
+ * 1. Unlike dev environment there’s no Sudo in Rococo, so we need to important a wallet first.
+ * a) Export a wallet from polkadot.js plugin, and save it as a file ./private/seed.json
+ * b) Run "PASS_PHRASE=<password_for_unlock> npm run mangata-rococo"
+ *
+ * 2. You need to have balances in both MGR and TUR on Mangata Rococo
+ * 3. You need to have balances in TUR on Turing Staging
+ */
+
+// Create a keyring instance
+const keyring = new Keyring({ type: 'sr25519' });
+
+/** * Main entrance of the program */
+async function main() {
+    await cryptoWaitReady();
+
+    console.log('Initializing APIs of both chains ...');
+    const turingHelper = new TuringHelper(TuringStaging);
+    await turingHelper.initialize();
+
+    const mangataHelper = new MangataHelper(MangataRococo);
+    await mangataHelper.initialize();
+
+    const turingChainName = turingHelper.config.key;
+    const mangataChainName = mangataHelper.config.key;
+    const turingNativeToken = _.first(turingHelper.config.assets);
+    const mangataNativeToken = _.first(mangataHelper.config.assets);
+
+    console.log(`\nTuring chain name: ${turingChainName}, native token: ${JSON.stringify(turingNativeToken)}`);
+    console.log(`Mangata chain name: ${mangataChainName}, native token: ${JSON.stringify(mangataNativeToken)}\n`);
+
+    console.log('1. Reading token and balance of account ...');
+
+    const json = await readMnemonicFromFile();
+    const keyPair = keyring.addFromJson(json);
+    keyPair.unlock(process.env.PASS_PHRASE);
+
+    const account = new Account(keyPair);
+    await account.init([turingHelper, mangataHelper]);
+    account.print();
+
+    const mangataAddress = account.getChainByName(mangataChainName)?.address;
+    const turingAddress = account.getChainByName(turingChainName)?.address;
+    const poolName = `${mangataNativeToken.symbol}-${turingNativeToken.symbol}`;
+
+    console.log('\n2. Add a proxy on Mangata for paraId 2114, or skip this step if that exists ...');
+
+    const proxyAddress = mangataHelper.getProxyAccount(mangataAddress, turingHelper.config.paraId);
+    const proxiesResponse = await mangataHelper.api.query.proxy.proxies(mangataAddress);
+    const proxies = _.first(proxiesResponse.toJSON());
+
+    const proxyType = 'AutoCompound';
+    const matchCondition = { delegate: proxyAddress, proxyType };
+
+    const proxyMatch = _.find(proxies, matchCondition);
+
+    if (proxyMatch) {
+        console.log(`Found proxy of ${account.address} on Mangata: `, proxyMatch);
+    } else {
+        if (_.isEmpty(proxies)) {
+            console.log(`Proxy array of ${account.address} is empty ...`);
+        } else {
+            console.log('Proxy not found. Expected', matchCondition, 'Actual', proxies);
+        }
+
+        console.log(`Adding a proxy for paraId ${turingHelper.config.paraId}. Proxy address: ${proxyAddress} ...`);
+        await mangataHelper.addProxy(proxyAddress, proxyType, account.pair);
```

Comment:
proxyType needs to be explicitly passed into addProxy() now. Providers other than mangataHelper also need to change.